_disability-benefits/claims-process/claim-types/predischarge-claim.md

```diff
@@ -4,24 +4,15 @@ source: http://www.benefits.va.gov/predischarge/index.asp
 ---
 
 <div class="main" role="main" markdown="0">
-
-<div class="action-bar">
-  <div class="row">
-    <div class="small-12 columns">
-      <a class="usa-button-primary va-button-primary" href="/disability-benefits/apply-for-benefits/">Apply for Disability Benefits</a>
-    </div>
-  </div>
-</div>
-
 <div class="section one" markdown="0">
 <div class="primary" markdown="0">
 <div class="row" markdown="0">
-<div class="small-12 medium-8 columns">
-
+<div class="small-12 columns" markdown="1">
 
 <div markdown="1">
 
 It is best to start the process of filing a disability claim while you are on active duty in predischarge status. Processing times tend to be faster for claims submitted before separation than for those submitted after separation. 
+>>>>>>> vre-master
```

Comment:
not sure if this stray commit was intentional @kantrowitz ? 

updating to be most current vre version.